packages/nvidia_nat_autogen/src/nat/plugins/autogen/callback_handler.py

```
@@ -0,0 +1,627 @@
+                    # Extract text from chunk if available
+                    try:
+                        if hasattr(chunk, "content") and chunk.content:
+                            output_chunks.append(str(chunk.content))
+                        # Check for usage in final chunk
+                        if hasattr(chunk, "usage") and chunk.usage:
+                            if hasattr(chunk.usage, "model_dump"):
+                                usage_payload = chunk.usage.model_dump()
+                            elif isinstance(chunk.usage, dict):
+                                usage_payload = chunk.usage
+                    except Exception:
+                        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)